### PR TITLE
feat: Five Binaries go-cardano-serialization

### PR DESCRIPTION
**Pull request by wolf31o2**

### CODE REVIEW

File: config/github_repos.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
   "emurgo/yoroi",
   "emurgo/yoroi-frontend",
   "emurgo/yoroi-mobile",
+  "github.com/fivebinaries/go-cardano-serialization",
```
Comment (robinboening) [inline config/github_repos.json]: @wolf31o2 Thanks for the contribution! Could you remove the domain part though? ❤️

Comment (wolf31o2) [reply config/github_repos.json]: D'oh. Yeah, I'll fix it, shortly. I got too excited.

Comment (robinboening) [reply config/github_repos.json]: Awesome, thanks :)

Comment (wolf31o2) [reply config/github_repos.json]: Fixed

**Pull request merged**